libminifi/include/core/TypedValues.h

```diff
@@ -163,7 +204,11 @@ class DataSizeValue : public TransformableValue, public state::response::UInt64V
 
     const char *cvalue = input.c_str();
     char *pEnd;
-    auto ival = std::strtoll(cvalue, &pEnd, 0);
+
+    T ival = 0;
+    if(!StringToNum(cvalue, &pEnd, ival)) {
```

Comment:
This is only applicable if we reach the null terminator and it is not a byte string

Reply:
Not sure I get the point, this does the same strto(u)ll call inside that was here before. 
The only change is the return value, which comes from AssignIfFits.